src/parser/GenericParser.test.ts

```diff
@@ -266,12 +289,12 @@ describe("GenericParser", () => {
 
     describe("parses TypeScript real lines of code metric", () => {
         it("should count correctly for a non-empty file, ignoring comments and empty lines", () => {
-            const inputPath = fs.realpathSync(tsTestResourcesPath + "real-lines-of-code.ts");
+            const inputPath = fs.realpathSync(phpTestResourcesPath + "php-example-code.php");
```

Comment:
I am confused by this change since the description is related to TypeScript :-)

Reply:
Ah, yes, that is wrong, thank you. It should be corrected now.

This long file starts to get confusing, especially when merge conflicts occur. I will split it into separate files one the "merging nightmare" #23 is merged (introducing "async" and "await" in every test case).